### ISSUE TITLE
navbar is not responsive  in the mobile screen

### ISSUE THREAD

**Issue by Abhishekvermaa** [OpenCodeEra/OpenCodeEra#59]
pls, assign this issue to me under  the hacktoberfest.

**Comment by AdarshRawat1** [comment]
@Abhishekvermaa Have you made any progress?

**Comment by Abhishekvermaa** [comment]
Yes I did the part of navbar responsiveness but there is some problem which I'm facing that whenever I trying to do navbar responsive background color change to white.
Can you help me out in this ??

**Comment by Vijay-Gunwant** [comment]
@AdarshRawat1 i would like to work on this issue

**Comment by arujjval** [comment]
@AdarshRawat1 pls assign this to me.
